### PR TITLE
Model predictions and features

### PR DESCRIPTION
**Pull request by Serafim179**
Branch with the fixed and experimental models using top-50 features.
Should be merged with master for data quality incorporation into model output distribution size.

### CODE REVIEW

File: modelling/uncertainty_model/model.py [hunk 1]
```diff
@@ -39,4 +175,127 @@ def no_feature_model(X):
...
+        begins.append(begin)
+        ends.append(end)
+
+    # if print_ci:
+    #     print('Confidence interval: ', begin, end)
+    #     print('Mean: ', np.mean([begin, end]))
+    #     print('True: ', y_test[sample_id])
+    #     print('Classification: ', (begin <= y_test[sample_id]) and (y_test[sample_id] <= end))
+
+    return np.array(begins), np.array(ends)
```
Comment (Serafim179) [inline modelling/uncertainty_model/model.py:268]: Tried to vectorize the output distribution sampling, but it somehow ended up being even slower, so I just left it as is and didn't use it.

File: test.py [hunk 2]
```diff
@@ -1,32 +1,19 @@
...
-    print('Confidence interval: ', begin, end)
-    print('Mean: ', np.mean([begin, end]))
-    print('True: ', y_test[sample_id])
-
-
-model = load_model('model.h5')
-#history = pickle.load(open('/trainHistoryDict'), "rb")
-
-X_test, y_test, fs = make_seq('preprocessed_data/nemcova_data/nemcova_data.csv', 'preprocessed_data/nemcova_data/ground_truths_nemcova.csv', return_features=True)
+    X_train, X_test, _, y_test, _, fs_test = train_test_split(X, y, feats, test_size=0.2, random_state=55)
```
Comment (gianlucatruda) [inline test.py:16]: Our dataset may be a bit small to get robust results from this. We probably need to do 5-fold CV

Comment (gianlucatruda) [reply test.py:16]: Although we're just evaluating confidence intervals.

**Review by gianlucatruda — APPROVED** [review]
Had a skim. Seems fine. Left some notes. You can merge when ready. Nice work 👍

**Pull request merged**